### PR TITLE
adding a new page for node operators that describes how to reclaim node disk space

### PR DESCRIPTION
**Pull request by vishalchangrani**
https://docs-git-vishal-nodeoperationreclaimdisk-onflow.vercel.app/networks/node-ops/node-operation/reclaim-disk

### CODE REVIEW

File: docs/networks/node-ops/node-operation/reclaim-disk.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Managing disk space
+description: How to manage the node disk space
+---
+
+As the chain advances, nodes receive chain data and store it on disk.
+Hence, the disk usage of a node keeps increasing gradually over time.
+
+In addition to this, currently nodes also experience intermittent spikes in disk usage caused by the compaction process of the baddger database used by the node software.
```
Comment (zhangchiqing) [inline docs/networks/node-ops/node-operation/reclaim-disk.md]:
```suggestion
In addition to this, currently nodes also experience intermittent spikes in disk usage caused by the compaction process of the badger database used by the node software.
```

File: docs/networks/node-ops/node-operation/reclaim-disk.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+3. If needed, please add more disk space to the node from time to time.
+
+> It highly recommended to setup alerting around disk usage to facilitate timely action and avoid any downtime and subsequent reward slashing for the node.
+
+
+## Reclaiming disk space
+
+### Access, Collection, Consensus and Verification node
```
Comment (peterargue) [inline docs/networks/node-ops/node-operation/reclaim-disk.md:29]: it's probably worth noteing that Access nodes will no longer be able to serve AccessAPI requests for data before the dynamic bootstrap height.

Comment (vishalchangrani) [reply docs/networks/node-ops/node-operation/reclaim-disk.md:29]: oh shoot - totally forgot to mention that ...updating.

**Review by sjonpaulbrown — APPROVED** [review]

File: docs/networks/node-ops/node-operation/reclaim-disk.md (hunk 2 above)
Comment (vishalchangrani) [reply docs/networks/node-ops/node-operation/reclaim-disk.md:29]: added a section - Limitation for access node

**Review by zhangchiqing — APPROVED** [review]

**Pull request merged**